src/documentCreator/services/token.ts

```diff
@@ -0,0 +1,64 @@
+import { TokenRegistry, WriteableTitleEscrowOwner, TitleEscrowOwner } from "@govtechsg/oa-token";
+import { WrappedDocument } from "@govtechsg/open-attestation";
+import { getData } from "@govtechsg/tradetrust-schema";
+import { getLogger } from "../logger";
+import { Wallet, ethers } from "ethers";
+import { get } from "lodash";
+
+const { trace } = getLogger("saga:tokenService");
+
+interface InitializeTokenInterface {
+  document: WrappedDocument<any>;
```

Comment:
```suggestion
  document: WrappedDocument<v2.OpenAttestation>;
```

Was asking @Nebulis  about this and I think since we are only dealing with v2 documents now, we can use this type signature first, gives us more types anway.